ca/ca.go

```diff
@@ -415,7 +414,7 @@ func (ca *CertificateAuthorityImpl) IssueCertificate(ctx context.Context, csr x5
 	serialBytes[0] = byte(ca.prefix)
 	_, err = rand.Read(serialBytes[1:])
 	if err != nil {
-		err = core.InternalServerError(err.Error())
+		err = berrors.InternalServerError("boulder/ca.IssueCertificate: failed to generate serial: %s", err)
```

Comment:
It seems like the `boulder/` prefix is redundant, is there a reason you decided to include it for the `InternalServerError` descriptions?